packages/core/data-transfer/lib/engine/index.ts

```diff
@@ -385,16 +385,37 @@ class TransferEngine<
     });
   }
 
-  async transferMedia(): Promise<void> {
-    const stageName: TransferStage = 'media';
+  async transferAssets(): Promise<void> {
+    const stageName: TransferStage = 'assets';
+    const inStream = await this.sourceProvider.streamAssets?.();
+    const outStream = await this.destinationProvider.getAssetsStream?.();
+
+    if (!inStream) {
+      throw new Error('Unable to transfer links, source stream is missing');
+    }
+
+    if (!outStream) {
+      throw new Error('Unable to transfer links, destination stream is missing');
+    }
```

Comment:
```suggestion
    const inStream = await this.sourceProvider.streamAssets?.();

    if (!inStream) {
      return;
    }

    const outStream = await this.destinationProvider.getAssetsStream?.();
    if (!outStream) {
      return;
    }
```

I think this is why the tests are failing. Streams aren't required anymore.

Reply:
oh, nice catch! Thanks